tests/migrate/fix_course_coach.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/* For licensing terms, see /license.txt */
+/**
+ * Set course coach if there is not assigned already one 
+ * @author Angel Fernando Quiroz Campos <angel.quiroz@beeznest.com>
+ * @package chamilo.migration
+ */
+if (PHP_SAPI != 'cli') {
+    exit;
+}
+
+require_once dirname(__FILE__) . '/../../main/inc/global.inc.php';
```

Comment:
__DIR__ is about 4 times faster than dirname(__FILE__) :-)
See https://beeznest.com/blog/2015/02/17/benchmark-dirname-vs-dir-php/